### ISSUE TITLE
abnormal lipogenesis

### ISSUE THREAD

**Issue by MonikaTomczuk** [obophenotype/mammalian-phenotype-ontology#2955]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
abnormal lipogenesis

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Process of fatty acid and triglyceride synthesis from glucose or other substrate or the metabolic formation of fat. 
From Merriam-Webster: the formation of fatty acids from acetyl coenzyme A in the living body and especially in adipose tissue and liver
https://www.merriam-webster.com/dictionary/lipogenesis
https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/lipogenesis

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
J:249664 distinguishes between lipogenesis and lipolysis using two models. We already have a lipolysis MP term. We also have 38 annotations using lipogenesis in the notes so it makes sense to add the term. 

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)
abnormal lipid homeostasis 

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
MP:0020866 placed under abnormal lipid metabolism as this follows the GO structure

**Comment by sbello** [comment]
MP:0020868 - decreased lipogenesis
MP:0020867 - increased lipogenesis
